packages/actor-query-source-identify-hypermedia-annotate-source/lib/ActorQuerySourceIdentifyHypermediaAnnotateSource.ts

```diff
@@ -0,0 +1,60 @@
+import type { MediatorMergeBindingsContext } from '@comunica/bus-merge-bindings-context';
+import type {
+  IActionQuerySourceIdentifyHypermedia,
+  IActorQuerySourceIdentifyHypermediaOutput,
+  IActorQuerySourceIdentifyHypermediaArgs,
+  IActorQuerySourceIdentifyHypermediaTest,
+  MediatorQuerySourceIdentifyHypermedia,
+} from '@comunica/bus-query-source-identify-hypermedia';
+import { ActorQuerySourceIdentifyHypermedia } from '@comunica/bus-query-source-identify-hypermedia';
+import { ActionContextKey } from '@comunica/core';
+import { QuerySourceAddSourceAttribution } from './QuerySourceAddSourceAttribution';
+
+/**
+ * A comunica None Query Source Identify Hypermedia Actor.
+ */
+export class ActorQuerySourceIdentifyHypermediaAnnotateSource extends ActorQuerySourceIdentifyHypermedia {
+  public readonly mediatorMergeBindingsContext: MediatorMergeBindingsContext;
+  public readonly mediatorQuerySourceIdentifyHypermedia: MediatorQuerySourceIdentifyHypermedia;
+
+  public constructor(args: IActorQuerySourceIdentifyHypermediaAnnotateSourceArgs) {
+    super(args, 'file');
+  }
+
+  public async testMetadata(
+    _action: IActionQuerySourceIdentifyHypermedia,
+  ): Promise<IActorQuerySourceIdentifyHypermediaTest> {
+    if (_action.context.get(KEY_CONTEXT_WRAPPED)) {
+      throw new Error('Unable to wrap link queues multiple times');
+    }
+    return { filterFactor: 0 };
+  }
+
+  public async run(action: IActionQuerySourceIdentifyHypermedia): Promise<IActorQuerySourceIdentifyHypermediaOutput> {
+    this.logInfo(action.context, `Wrapped source with provenance: ${action.url}`);
```

Comment:
Not sure if we really need this logger here.
If you feel strongly about it, I'd lower it to the debug-level.

Reply:
I just copied it from another hypermedia actor and figured I would keep it. I'll remove it